.travis.yml

```diff
@@ -1,20 +0,0 @@
-language: java
-
-jdk:
- - openjdk8
- - openjdk11
- - openjdk15
-
-git:
-  quiet: true
-
-install:
-  - ./mvnw install -B -V -DskipTests -Dair.check.skip-all
-  - ./mvnw dependency:go-offline -B -Dsilent
```

Comment:
I skipped this since it's wasteful to do. It downloads EVERYTHING referenced by the pom even if it's not going to be used. It's useful if you want to cache things but a waste of time otherwise.